### ISSUE TITLE
Generated App: 500 Internal server error is returned when a required parent ID is missing

### ISSUE THREAD

**Issue by yuval-hazaz** [amplication/amplication#696]
Running the following query on the sample application returns 500 error. 

```
curl --location --request POST 'https://ckgphg6vt545610s6vxradkwr-server-ahjgcbl6pa-ue.a.run.app/projects' \
--header 'Content-Type: application/json' \
--header 'Authorization: Basic YWRtaW46YWRtaW4=' \
--data-raw '{
    "name":"my project name"
}'
```

The error happens because the request is missing a required user ID.
This error appears in the log 
```
Error: Null constraint violation on the fields: (`userId`)
```

An error on missing a required field is expected on the client

**Comment by yenicelik** [comment]
probably related to this as much as i understand: https://github.com/amplication/amplication/issues/1387 ?

**Comment by yuval-hazaz** [comment]
Already resolved in v0.0.6
